components/cover/basic/src/index.scss

```diff
@@ -10,16 +10,16 @@
     }
   }
 
-  max-height: $mah-cover-basic;
+  height: $mah-cover-basic;
```

Comment:
Are you sure of this? the behavior changes and max-height works in IE.
No need to answerd. Just want to call your attention on it

Reply:
yep